src/sflow_plugin/sflow_collector.cpp

```diff
@@ -41,6 +41,9 @@ bool sflow_lua_hooks_enabled = false;
 std::string sflow_lua_hooks_path = "/usr/src/fastnetmon/src/sflow_hooks.lua";
 #endif
 
+// Ethertype of outer tag in QinQ
+uint16_t sflow_qinq_ethertype = 0x8100;
```

Comment:
I think we also need explicit flag to enable processing for QnQ traffic. FastNetMon does not handle it now and after update it will start processing QnQ traffic. 

Let's add one more variable like "process_qnqn_sflow" disabled by default but enabled in default configuration.

Reply:
Is there really a need for this flag? Now FNM correctly handles both double and single vlan tagging.

Reply:
Yes, that's kind of mandatory because people may rely on fact that FastNetMon cannot decode QnQ traffic.